doc/045_working_with_repos.rst

```diff
@@ -252,3 +257,13 @@ check all repository pack files over 5 separate invocations:
     $ restic -r /srv/restic-repo check --read-data-subset=3/5
     $ restic -r /srv/restic-repo check --read-data-subset=4/5
     $ restic -r /srv/restic-repo check --read-data-subset=5/5
+
+Use ``--read-data-subset=n%`` to check a randomly choosen subset of the
+repository pack files. It takes one parameter, ``n``, the percentage of pack
+files to check as a floating point number. This will not guarantee to cover all
```

Comment:
Should probably write "as an integer or floating point number" to make it clear that both x% and x.y% are accepted.

Reply:
Fixed in https://github.com/restic/restic/pull/3038/commits/4943b1ebda8ad41c5cd485e202336e780a1e8c77